src/parser/avc_defines.h

```diff
@@ -33,7 +33,7 @@ THE SOFTWARE.
 
 #define AVC_MAX_REF_FRAME_NUM                           16
 #define AVC_MAX_REF_PICTURE_NUM                         32
-#define AVC_MAX_DPB_FRAMES                              16
+#define AVC_MAX_DPB_FRAMES                              18
```

Comment:
why do we need to increase this to 18?

Reply:
Due to current design limitations. I can explain in the call.